<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChallengePreview</class>
 <widget class="QDialog" name="ChallengePreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Демонстрация</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbl_challenge_list">
     <property name="text">
      <string>Список испытаний:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tbl_challenges"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_start">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Старт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_pause">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Пауза</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_stop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Стоп</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
